stripe/src/main/java/com/stripe/android/paymentsheet/BaseAddCardFragment.kt

```diff
@@ -277,6 +280,33 @@ internal abstract class BaseAddCardFragment(
                 )
             }
         cardMultilineWidget.postalCodeErrorListener = null
+
+        billingAddressView.postalCodeViewListener =
+            object : BillingAddressView.PostalCodeViewListener {
+                override fun onLosingFocus(country: Country?, isPostalValid: Boolean) {
```

Comment:
Can this logic be in the BIllingAddressView?  It feels like anyone that uses this view is going to need this same logic.

Reply:
the listener instance needs to access `BaseAddCardFragment#billingErrors`. We do define the `PostalCodeViewListener` interface in `BillingAddressView` though

Reply:
It looks like you added billingErrors to the BaseAddCardFragment because that is where cardErrors (the equivalent for the MultilineWidget is located.   It makes sense to keep it consistent for this PR.  

Were we talking about going forward though keeping this logic inside of the widgets?

Reply:
Good idea, unfortunately for this specific case it's not quite easy as we're using `CardMultilineWidget` for card errors and `BillingAddressView` for billing errors - however if both card and billing are in the same widget, we can use a single view to toggle them, I actually added this to the [CardFormView](https://paper.dropbox.com/doc/API-Proposal-CardFormView--BIRF8cbzWTKUeD87PFo_QkobAg-w2duapoMsbJjSfPTDtxKe) doc